pkg/koordlet/qosmanager/framework/context.go

```diff
@@ -62,20 +62,22 @@ func (r *Evictor) Start(stopCh <-chan struct{}) error {
 
 func (r *Evictor) EvictPodsIfNotEvicted(evictPods []*corev1.Pod, node *corev1.Node, reason string, message string) {
 	for _, evictPod := range evictPods {
-		r.evictPodIfNotEvicted(evictPod, node, reason, message)
+		r.EvictPodIfNotEvicted(evictPod, node, reason, message)
 	}
 }
 
-func (r *Evictor) evictPodIfNotEvicted(evictPod *corev1.Pod, node *corev1.Node, reason string, message string) {
+func (r *Evictor) EvictPodIfNotEvicted(evictPod *corev1.Pod, node *corev1.Node, reason string, message string) bool {
```

Comment:
please add some UTs for an exposed method

Reply:
OK

Reply:
I failed to add a test which evict failed because of serviceaccount has no permission on pod/eviction api.
Help

Reply:
A UT is enough, not an E2E. So I suppose the test should not require a role with `pod/eviction`. The kubelet eviction service can be mocked in that UT.

Reply:
OK